t/Test15-regrid_land.sh

```diff
@@ -61,10 +44,30 @@
 		--input_file 00050101.land_static  \
 		--scalar_field soil_frac,lake_frac,glac_frac,area,soil_area,lake_area,glac_area  \
 		--output_file out.nc  \
-		--remap_file remap_file.nc 
-  [ "$status" -eq 0 ] 
+		--remap_file remap_file.nc
+  [ "$status" -eq 0 ]
+
+# parallel call
+  if [ -z "$skip_mpi" ]; then
```

Comment:
@rem1776 @ngs333 This parallel test runs (and fails) when the tests are run interactively.

Are we running the parallel tests anywhere? We probably should be, if we can without too much trouble.